sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawNetwork.java

```diff
@@ -1,6 +1,28 @@
 package com.mesosphere.sdk.specification.yaml;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
 /**
  * Raw YAML network. Null class for now, since only network name is supported, but will gain fields in the future.
  */
-public class RawNetwork { }
+public class RawNetwork {
+    private final List<Integer> hostPorts;
+    private final List<Integer> containerPorts;
+
+    private RawNetwork(
+            @JsonProperty("host-ports") List<Integer> hostPorts,
+            @JsonProperty("container-ports") List<Integer> containerPorts) {
+        this.hostPorts = hostPorts;
+        this.containerPorts = containerPorts;
+    }
+
+    public List<Integer> getHostPorts() { return hostPorts; }
+
+    public List<Integer> getContainerPorts() { return containerPorts; }
+
+    public Integer numberOfPortMappings() {
+        return Math.min(hostPorts.size(), containerPorts.size());
```

Comment:
Will this fail silently if the lengths of the two are different? As a framework writer I want to be yelled at if I mess this up as early as possible in the process.